dashboard/settings.py

```diff
@@ -449,6 +449,12 @@ def apply_env_overrides(env: Dict[str, Any], environ: os._Environ) -> Dict[str,
 # Only need view permission for exports
 IMPORT_EXPORT_EXPORT_PERMISSION_CODE = 'view'
 
+# Change the default project ID for BigQuery if needed (This is typically the one that quotas are run against and logged into)
+DEFAULT_PROJECT_ID = ENV.get("DEFAULT_PROJECT_ID", None)
+
+# Override the default project ID for BigQuery if needed, like to unizin-shared
+DATASET_PROJECT_ID = ENV.get("DATASET_PROJECT_ID", None)
```

Comment:
Should the default value be DEFAULT_PROJECT_ID, instead of None?

Reply:
Maybe, but then we might need a new variable to indicate whether or not this is running on synthetic data or not. 

Maybe it's better to make that explicit. I'm not sure yet, this still has some more work on it.